fxa/oauth.py

```diff
@@ -239,11 +240,21 @@ def verify_token(self, token, scope=None):
             resp = None
 
         if resp is None:
-            url = '/verify'
-            body = {
-                'token': token
+            ## TODO we want to fetch https://oauth.accounts.firefox.com/.well-known/openid-configuration and then get the jwks_uri key to get the following url, but we'll just hardcode it like this for now until we are closer to production. We need to do this because oauth.accounts.firefox.com does not yet respond to .well-known, but accounts.firefox.com does; for consistency, oauth.accounts.firefox.com should respond to .well-known. 
+            
+            secret = jwt.algorithms.RSAAlgorithm.from_jwk(
+                self.apiclient.get('/v1/jwks')
+            )
+            decoded = jwt.decode(
+                token, secret, algorithms=['RS256'], options={'verify_aud': False}
+            )
+            resp = {
+                'user': decoded['sub'],
+                'client_id': decoded['client_id'],
+                'scope': decoded['scope'],
+                'generation': decoded['fxa-generation'],
+                'profile_changed_at': decoded['fxa-profileChangedAt']
```

Comment:
nit: the client code here generally tries to be defensive against server misbehaviour, so it may be worth using `.get()` rather than `[]` for some of these fields. Alternately, you could wrap this whole block in a try-except to catch any unexpected `KeyError` and turn it into an `OutOfProtocolError` (see the code just below this for an example of `OutOfProtocolError`.